browser.js

```diff
@@ -212,20 +212,19 @@ function zoomInit() {
 	const style = document.createElement('style');
 	style.id = 'zoomFactor';
 
+	// enable OS specific styles
+	document.documentElement.classList.add(`os-${process.platform}`);
+
 	document.body.appendChild(style);
 	setZoom(zoomFactor);
+
+	// activate Dark Mode if it was set before quitting
+	setDarkMode();
 }
 
 document.addEventListener('DOMContentLoaded', () => {
 	zoomInit();
 
-	// enable OS specific styles
-	document.documentElement.classList.add(`os-${process.platform}`);
```

Comment:
This shouldn't be moved. There's nothing here that needs React being ready, just the DOM.